autonormalize/tests/test_normalize.py

```diff
@@ -178,3 +183,223 @@ def test_make_indexes():
+    assert normalized_entityset.entities[0].variable_types['jersey_num'] == Numeric
+    assert normalized_entityset.entities[0].variable_types['player_name'] == Categorical
+
+    assert normalized_entityset.entities[1].variable_types['team'] == Index
+    assert normalized_entityset.entities[1].variable_types['city'] == Id
+
+    assert normalized_entityset.entities[2].variable_types['city'] == Index
+    assert normalized_entityset.entities[2].variable_types['state'] == SubRegionCode
+
+
+def test_auto_entityset_default_args():
+    dic = {'team': ['Red', 'Red', 'Red', 'Orange', 'Orange', 'Yellow',
+                    'Yellow', 'Green', 'Green', 'Blue'],
+           'jersey_num': [1, 2, 3, 1, 2, 1, 5, 8, 2, 2],
+           'player_name': ['A', 'B', 'C', 'D', 'A', 'E', 'B', 'A', 'G', 'H'],
+           'city': ['boston', 'boston', 'boston', 'chicago', 'chicago',
+                    'honolulu', 'honolulu', 'boston', 'boston', 'austin'],
+           'state': ['US-MA', 'US-MA', 'US-MA', 'US-IL', 'US-IL', 'US-HI', 'US-HI', 'US-MA', 'US-MA', 'US-TX']}
+    df = pd.DataFrame(dic)
+    normalized_entityset = autonormalize.auto_entityset(df)
+
+    dic_one = {'jersey_num_team': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+               'team': ['Red', 'Red', 'Red', 'Orange', 'Orange', 'Yellow',
+                        'Yellow', 'Green', 'Green', 'Blue'],
+               'jersey_num': [1, 2, 3, 1, 2, 1, 5, 8, 2, 2],
+               'player_name': ['A', 'B', 'C', 'D', 'A', 'E', 'B', 'A', 'G', 'H']}
+
+    dic_two = {'team': ['Blue', 'Green', 'Orange', 'Red', 'Yellow'],
+               'city': ['austin', 'boston', 'chicago', 'boston', 'honolulu']}
+
+    dic_three = {'city': ['austin', 'boston', 'chicago', 'honolulu'],
+                 'state': ['US-TX', 'US-MA', 'US-IL', 'US-HI', ]}
+
+    assert len(normalized_entityset.entities) == 3
+
+    assert normalized_entityset.entities[0].df.equals(pd.DataFrame(dic_one))
+    assert normalized_entityset.entities[1].df.equals(pd.DataFrame(
+        dic_two, index=['Blue', 'Green', 'Orange', 'Red', 'Yellow']))
+    assert normalized_entityset.entities[2].df.equals(pd.DataFrame(
+        dic_three, index=['austin', 'boston', 'chicago', 'honolulu']))
+
+    assert normalized_entityset.entities[0].variable_types['jersey_num_team'] == Index
+    assert normalized_entityset.entities[0].variable_types['team'] == Id
+    assert normalized_entityset.entities[0].variable_types['jersey_num'] == Numeric
+    assert normalized_entityset.entities[0].variable_types['player_name'] == Categorical
+
+    assert normalized_entityset.entities[1].variable_types['team'] == Index
+    assert normalized_entityset.entities[1].variable_types['city'] == Id
+
+    assert normalized_entityset.entities[2].variable_types['city'] == Index
+    assert normalized_entityset.entities[2].variable_types['state'] == Categorical
+
+
+def test_auto_entityset_custom_args():
+    dic = {'team': ['Red', 'Red', 'Red', 'Orange', 'Orange', 'Yellow',
```

Comment:
these dictionaries get re-used in several tests, creating many duplicated lines of code.  Could you turn these dictionaries into pytest fixtures so they're only defined once?